assets/chezmoi.io/docs/user-guide/manage-machine-to-machine-differences.md

```diff
@@ -55,16 +54,13 @@ You can then open the template (which will be saved in the file
 $ chezmoi edit ~/.gitconfig
 ```
 
-The file should look something like:
+Edit the to file should look something like:
```

Comment:
Incomplete?

Reply:
Thanks! Fix in 16e5084aac931a4433e4e9f2844412bd2d270a68.